src/Particle/ParticleSet.h

```diff
@@ -285,26 +294,23 @@ class ParticleSet
   }
 
   /** update the internal data
-   *@param iflag index for the update mode
+   *@param skip SK update if skipSK is true
    */
-  void update(int iflag=0);
+  void update(bool skipSK=false);
```

Comment:
update() is not supposed to be called often.
donePbyP() computes SK before any estimators.